pkg/operator/hive/hive_controller.go

```diff
@@ -246,3 +255,25 @@ func (r *ReconcileHiveConfig) Reconcile(request reconcile.Request) (reconcile.Re
 
 	return reconcile.Result{}, nil
 }
+
+func (r *ReconcileHiveConfig) deleteLegacyComponents(hLog log.FieldLogger) error {
+	// Ensure legacy hiveadmission Daemonset is deleted, we switched to a Deployment:
+	// TODO: this can be removed once rolled out to opshive, our only persistent environment.
```

Comment:
Once rolled out to hive-stage and hive-prod. (opshive won't be getting any more updates)